Tests/DatadogTests/Datadog/CrashReporting/CrashReporterTests.swift

```diff
@@ -12,7 +12,7 @@ class CrashReporterTests: XCTestCase {
     // MARK: - Sending Crash Report
 
     func testWhenPendingCrashReportIsFound_itIsSentAndPurged() throws {
-        let expectation = self.expectation(description: "`LoggingOrRUMsender` sends the crash report")
+        let expectation = self.expectation(description: "`LoggingAndRUMsender` sends the crash report")
```

Comment:
nit/ it seems to be an artifact of V2 refactoring - neither `LoggingOrRUMsender` nor `LoggingAndRUMsender` do exist and instead we use `CrashReportSender`.
```suggestion
        let expectation = self.expectation(description: "`CrashReportSender` sends the crash report")
```